jetty11/src/test/java/ch/qos/logback/access/jetty/JettyBasicTest.java

```diff
@@ -68,7 +68,7 @@ public void getRequest() throws Exception {
 
     @Test
     public void eventGoesToAppenders() throws Exception {
-        URL url = new URL(JETTY_FIXTURE.getUrl());
+        URL url = new URL("http://localhost:" + RANDOM_SERVER_PORT + "/path/foo%20bar;param?query#fragment");
```

Comment:
the same as [above](https://github.com/qos-ch/logback-access/pull/7/files#r1568407732)